### ISSUE TITLE
Can I pass my user agent in a script?

### ISSUE THREAD

**Issue by pbezpal** [artilleryio/artillery#873]
I want use my user-agent. Can i pass my user-agent in scenario

My user-agent
"user-agent":"Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/81.0.4044.138 YaBrowser/20.4.3.268 (beta) Yowser/2.5 Safari/537.36"

**Comment by christiansaiki** [comment]
Just add the user agent into the header
https://artillery.io/docs/guides/guides/http-reference.html#Setting-Headers

**Comment by hassy** [comment]
Yep, exactly what @christiansaiki said. You can go one step further and pass in a randomised user agent with a CSV file containing several user agents: https://artillery.io/docs/guides/guides/test-script-reference.html#Payload-Files
